### ISSUE TITLE
Accessibility: text color in Find and Mark dialog status bar fails to change with system visual theme settings

### ISSUE THREAD

**Issue by aev-mambro2** [notepad-plus-plus/notepad-plus-plus#5087]
### Description of the Issue

Accessibility: text color in Find and Mark dialog status bar fails to change with system visual theme settings. The text color stays blue no matter which visual theme I choose. Windows has been shipped with a high-contrast-black visual theme since I can remember. I'm using that theme, because of the state of my vision. It features bright colors for hyperlinks and text, and dark colors for backgrounds. The color on the status bar of the Find and Match dialog, that displays how many matches were found, fails to read any of the color settings from the Windows visual theme. It stays dark blue. That's a problem: dark blue against black / dark gray is unreadable. The status text might as well not exist at all. 

### Steps to Reproduce the Issue

1. Set Windows Visual Theme to High Contrast Black. 
2. Open Notepad++ and load any document. 
3. Open the Mark dialog and mark any word that exists in the opened document.

### Expected Behavior

I expect the amount of matches to show up in the dialog status bar. I expect to see that text in one of the bright text colors set by the Windows Visual Theme, just like its background is set to a dark color from the Windows Visual Theme.

### Actual Behavior

The text color of the match count in the dialog status bar stays dark blue. 

### Debug Information

Notepad++ v7.5.9   (32-bit)
Build time : Oct 14 2018 - 15:02:52
Path : C:\Program Files (x86)\Notepad++\notepad++.exe
Admin mode : OFF
Local Conf mode : OFF
OS : Windows 10 (64-bit)
Plugins : DSpellCheck.dll mimeTools.dll NppExport.dll ComparePlugin.dll JSMinNPP.dll NppColumnSort.dll NppConverter.dll PluginManager.dll PoorMansTSqlFormatterNppPlugin.dll XMLTools.dll 

Included a screen shot.
![npp-mark-dialog-shows-number-of-matches-in-hardcoded-blue-accessibility-20181130t1135](https://user-images.githubusercontent.com/42442483/49302526-5cdb9900-f495-11e8-94c2-5028b6126ec0.png)

**Comment by sasumner** [comment]
Very nice description of the issue.  This blue text can at times also be colored green or red--do these cases appear acceptably when you see them, or are they a problem too?

**Comment by sasumner** [comment]
@aev-mambro2 I'm not a developer on Notepad++, thus I have no control over what happens with this, but the solutions you describe might be prohibitive for a developer with limited time to implement.  

Thinking about it, it seems that red text color is currently used to indicate an error condition (text NOT found, invalid regular expression, perhaps some others) whereas green is used for success-but-informational conditions (where a match occurs after a wrap-around at an end of the file, perhaps others).

Since blue seems to result in reporting a match count, perhaps this could be considered also success-but-informational condition, and could be converted over to showing in green.  Do you think that is an acceptable compromise?

**Comment by aev-mambro2** [comment]
@sasumner  If the developers do decide that the good solution takes too much time, then yes, as a temporary compromise I wouldn't mind if the match count text gets displayed in the current green, and the no-match count text in the current red. 
  
But I'm not looking for a solution that helps just me. The solution I advised helps everyone. Thus I would prefer it if the developers could find time to implement that one. 
  
Since neither you nor me are Notepad++ developers, wouldn't you agree that continuing this line of discussion is moot?

**Comment by sasumner** [comment]
What makes it _moot_?  I think it is _valuable_.  The way it works is:  issues are opened (by anyone), issues are discussed/debated (by anyone), issues (are possibly) implemented (by a developer).  If a developer picks it up to work on, they have the prior discussion logged here as a reference during implementation.  

A developer can think, well, here's a nice-to-have solution that I maybe don't have time to do, but here's a couple of people that have agreed a lesser solution (perhaps as little of a change as one line of code) makes sense--I can do that one, get this issue closed, and make someone acceptably pleased.

Of course, I don't _know_ that is what would happen, but it seems a logical guess.

My observation is that Notepad++ developers seem to gravitate towards working on features that are important to themselves as users of Notepad++.  Since they seem to like the blue color (they implemented it, after all), and likely have no accessibility issues, how important would something like this be to them?  I'll let you consider that.  :-)

**Comment by alankilborn** [comment]
Similar request: https://community.notepad-plus-plus.org/topic/26063
